src/coreclr/src/jit/emit.cpp

```diff
@@ -5085,7 +5090,12 @@ unsigned emitter::emitEndCodeGen(Compiler* comp,
             }
             else
             {
-                printf("\nG_M%03u_IG%02u:\n", emitComp->compMethodID, ig->igNum);
+                printf("\nG_M%03u_IG%02u:", emitComp->compMethodID, ig->igNum);
+                if (emitComp->opts.disAddr)
```

Comment:
Maybe instead of `emitComp->opts.disAddr` this could be `!emitComp->opts.disDiffable` so it's on all the time?